Tools/Create-Azure-Sentinel-Solution/V2/createSolutionV2.ps1

```diff
@@ -0,0 +1,2169 @@
+                                    }
+                                    else {
+                                        [PSCustomObject] @{
+                                            name         = "playbook$playbookCounter-$paramName";
+                                            type         = "Microsoft.Common.TextBox";
+                                            label        = "$(PascalSplit $paramName)";
+                                            defaultValue = $defaultParamValue;
+                                            toolTip      = "Please enter $(if($paramName.IndexOf("-") -ne -1){$paramName}else{PascalSplit $paramName})";
+                                            constraints  = [PSCustomObject] @{
+                                                required          = $true;
+                                                regex             = "[a-z0-9A-Z]{1,256}$";
+                                                validationMessage = "Please enter the $(PascalSplit $paramName)"
+                                            }
+                                        }
+                                    }
+                                )
+                                $baseCreateUiDefinition.parameters.steps[$currentStepNum].elements[$baseCreateUiDefinition.parameters.steps[$currentStepNum].elements.Length - 1].elements += $playbookParamObject
+                                $defaultValue = $(if ($defaultParamValue) { $defaultParamValue } else { "" })
+                                $baseMainTemplate.parameters | Add-Member -NotePropertyName "playbook$playbookCounter-$paramName" -NotePropertyValue ([PSCustomObject] @{
+                                        defaultValue = $defaultValue;
+                                        type         = "string";
+                                        minLength    = 1;
+                                    })
+                            }
+                            $baseCreateUiDefinition.parameters.outputs | Add-Member -NotePropertyName "playbook$playbookCounter-$paramName" -NotePropertyValue "[steps('playbooks').playbook$playbookCounter.playbook$playbookCounter-$paramName]"
+                        }
+
+                        foreach ($playbookVariable in $playbookData.variables.PsObject.Properties) {
+                            $variableName = $playbookVariable.Name
+                            $variableValue = $playbookVariable.Value
+                            if ($variableValue -is [System.String]) {
+                                $variableValue = $(node "$PSScriptRoot/templating/replacePlaybookParamNames.js" $variableValue $playbookCounter)
+                            }
+                            if (($solutionName.ToLower() -eq "cisco meraki") -and ($variableName.ToLower().contains("apikey"))) {
+                                $baseMainTemplate.variables | Add-Member -NotePropertyName "playbook-$variableName" -NotePropertyValue "[$variableValue]"
```

Comment:
Is there any possibility to configure values like "cisco meraki" instead of hard coding here